src/commands/config/create.ts

```diff
@@ -0,0 +1,180 @@
+        fund: "ropsten",
+      };
+      walletPath = await CreatedWalletPath(createWalletParams);
+      info(`Wallet created at ${walletPath}`);
+    }
+    const walletFilePath = walletPath ? walletPath : args.encryptedWalletPath;
+    const wallet = await readFile(walletFilePath);
+    const walletObject = JSON.parse(wallet);
+
+    const configFile: ConfigFile = ConfigTemplate;
+    configFile.wallet = wallet;
+
+    const template = JSON.parse(await readFile(args.configTemplatePath));
+    const formsInTemplate = template.forms;
+    let documentStoreAddress = "";
+    let tokenRegistryAddress = "";
+    let verifiableDocumentDnsName = "";
+    let tokenRegistryDnsName = "";
+    const typeOfDocuments = formsInTemplate.map((formTemplate: any) => formTemplate.type);
+    if (typeOfDocuments.includes("VERIFIABLE_DOCUMENT")) {
+      info(`Enter password to continue deployment of Document Store`);
+      const deployDocumentStoreParams = {
+        encryptedWalletPath: walletFilePath,
+        network: "ropsten",
+        gasPriceScale: 1,
+        dryRun: false,
+        storeName: "Document Store",
+      };
+      const documentStore = await deployDocumentStore(deployDocumentStoreParams);
+      documentStoreAddress = documentStore.contractAddress;
+      success(`Document store deployed, address: ${highlight(documentStoreAddress)}`);
+      const DocumentStoreTemporaryDnsParams = {
+        networkId: 3,
+        address: documentStoreAddress,
+        sandboxEndpoint: "https://sandbox.openattestation.com",
+      };
+      info(`Creating temporary DNS for verifiable documents`);
+      verifiableDocumentDnsName = (await CreateTemporaryDns(DocumentStoreTemporaryDnsParams)) || "";
+    }
+    if (typeOfDocuments.includes("TRANSFERABLE_RECORD")) {
+      info(`Enter password to continue deployment of Token Registry`);
+      const deployTokenRegistryParams = {
+        registryName: "Token Registry",
+        registrySymbol: "DTR",
+        encryptedWalletPath: walletFilePath,
+        network: "ropsten",
+        gasPriceScale: 1,
+        dryRun: false,
+      };
+      const tokenRegistry = await deployTokenRegistry(deployTokenRegistryParams);
+      tokenRegistryAddress = tokenRegistry.contractAddress;
+      success(`Token registry deployed, address: ${highlight(tokenRegistryAddress)}`);
+      info(`Creating temporary DNS for transferable records`);
+      const TokenRegistryTemporaryDnsParams = {
+        networkId: 3,
+        address: tokenRegistryAddress,
+        sandboxEndpoint: "https://sandbox.openattestation.com",
+      };
+      tokenRegistryDnsName = (await CreateTemporaryDns(TokenRegistryTemporaryDnsParams)) || "";
+    }
+
+    formsInTemplate.forEach((form: any) => {
+      let updatedIssuers;
+      const updatedForm = form;
+      if (form.type === "VERIFIABLE_DOCUMENT") {
+        updatedIssuers = form.defaults.issuers.map((issuer: any) => {
+          if (issuer.identityProof.type === "DNS-DID") {
```

Comment:
dns creation does not work for DID here, we can eventually skip the did part for now

Reply:
ok fixed.